src/components/Navigation/Navigation.jsx

```diff
@@ -45,8 +45,8 @@ export default class Navigation extends React.Component {
               <Dropdown
                 className="navigation__languages"
                 items={[
-                  { title: 'English', url: 'https://webpack.js.org/' },
-                  { title: '中文', url: 'https://doc.webpack-china.org/' }
+                  { lang: 'en', title: 'English', url: 'https://webpack.js.org/' },
```

Comment:
What is the reason of this change?

Reply:
It's my mistake. This shouldn't be there. Thanks for taking a look :)